Payments/UserInterface/08a_Connections.md

```diff
@@ -0,0 +1,201 @@
+[!!Establish connections to a payment service provider](../UserInterface/02a_ListPaymentsAndCaptures.md)
```

Comment:
Check bitte den Link und den Namen, ich meine, die stimmen nicht.
Meinst du Manage connections to payment service providers (PSP)?